service/web/feed/handler.go

```diff
@@ -2,48 +2,84 @@ package feed
 
 import (
 	"context"
+	"strconv"
+	"time"
+	bizConstant "toktik/constant/biz"
+	bizConfig "toktik/constant/config"
+	"toktik/kitex_gen/douyin/feed"
+	feedService "toktik/kitex_gen/douyin/feed/feedservice"
+	"toktik/logging"
+	"toktik/service/web/mw"
+
 	"github.com/cloudwego/hertz/pkg/app"
-	"github.com/cloudwego/hertz/pkg/protocol/consts"
+	httpStatus "github.com/cloudwego/hertz/pkg/protocol/consts"
 	"github.com/cloudwego/kitex/client"
 	consul "github.com/kitex-contrib/registry-consul"
-	"log"
-	"toktik/constant/config"
-	"toktik/kitex_gen/douyin/feed"
-	feedService "toktik/kitex_gen/douyin/feed/feedservice"
+	"github.com/sirupsen/logrus"
 )
 
 var feedClient feedService.Client
 
 func init() {
-	r, err := consul.NewConsulResolver(config.EnvConfig.CONSUL_ADDR)
+	r, err := consul.NewConsulResolver(bizConfig.EnvConfig.CONSUL_ADDR)
 	if err != nil {
-		log.Fatal(err)
+		logging.Logger.WithError(err).Fatal("init feed client failed")
+		panic(err)
 	}
-	feedClient, err = feedService.NewClient(config.FeedServiceName, client.WithResolver(r))
+	feedClient, err = feedService.NewClient(bizConfig.FeedServiceName, client.WithResolver(r))
 	if err != nil {
-		log.Fatal(err)
+		logging.Logger.WithError(err).Fatal("init feed client failed")
+		panic(err)
 	}
 }
 func Action(ctx context.Context, c *app.RequestContext) {
+	methodFields := logrus.Fields{
+		"time":   time.Now(),
+		"method": "FeedAction",
+	}
+	logger := logging.Logger.WithFields(methodFields)
+	logger.Debugf("Process start")
+
 	latestTime := c.Query("latest_time")
-	actorId := c.GetUint32("user_id")
+	if _, err := strconv.Atoi(latestTime); err != nil {
```

Comment:
`lastest_time` is optional
![image](https://user-images.githubusercontent.com/53544726/218452028-9d66215d-fcc3-40ed-8a6e-72143147d19e.png)